heroku/cli
# refactor(certs-v5): move certs:auto:disable to CLI & upgrade to oclif

### Description

This converts `certs:auto:disable` to oclif and updates its associated test.

### Testing

1) Run `./bin/run certs:auto:disable -a APP` with this branch pulled down with an app that has acm enabled (please use a test app, lol)
2) Verify the command succeeds, requiring a confirmation step, and acm is disabled
3) Repeat step 1, this time using `--confirm` flag
4) Very command succeeds without confirmation step, and acm is disabled
5) Compare the results with prod cli command and make sure things look 👍
